### DISCUSSION TITLE
Concerning the SECRET parameter

### DISCUSSION THREAD

**Discussion by criticalcake** [umputun/remark42#861] [Q&A] [answered]
Hi there,

just a simple question concerning the SECRET parameter:
This parameter is intended to go with any hard to guess string.
The string does not need to match with any other string or key elsewhere.
E.g. it`s not used for authentication or anything similar.
Correct?

**Comment by umputun** [comment, marked as answer]
The secret used to sign JWT and should be a random, long, hard-to-guess string. For example, `unmaze-mirror-phu-yep-alogy-wirra-harr` will do or `TLbWnWr6KF5k0XT15` or anything like this.

This string shouldn't match anything else, neither auth parameter nor any other parameter. I usually recommend keeping the secret's backup in someplace, but in this case, it doesn't really matter. If you lose your secret and replace with a completely different one nothing will break, and the only side effect will be a rejection of all currently logged in user (will force them to authenticate again).

_Pls don't use provided examples; they are just this - examples._

**Comment by criticalcake** [comment]
@umputun thank you for clarifying this aspect to me. 👍 
And also: Thank you very much for providing remark42 as open source AND for your effort to support users like me! 🙏
